bonding-rose.lic

```diff
@@ -0,0 +1,114 @@
+      bput('stow pestle', 'You put')
+    end
+    unless exists?('mixing stick')
+      order_item(8860, 5)
+      bput('stow mix stick', 'You put')
+    end
+    unless exists?('small jar')
+      buy_item(19076, 'small jar')
+      bput('get oil in my small jar', 'You get')
+      bput('drop my oil', 'You drop')
+      bput('stow my small jar', 'You put')
+    end
+    buy_item(19076, 'oil')
+    fput('prep bless')
+    pause 8
+    fput('cast my oil')
+    bput('put my oil in my small jar', 'You put')
+    buy_item(19076, 'dracon crystals')
+    bput('put my crystals in my mortar', 'You put', 'The mortar already has')
+    bput('get my mortar', 'You get')
+    bput('get my pestle', 'You get')
+    while bput('crush crystals in my mortar with my pestle', 'into some dracon powder', 'with your pestle') == 'with your pestle'
+      pause 1
+    end
+    EquipmentManager.new.empty_hands
+    bput('get powder from my mortar', 'You get')
+    bput('break my powder', 'You break')
+    bput('drop my powder', 'You drop')
+    bput('break my powder', 'You break')
+    bput('drop my powder', 'You drop')
+#   This leaves the 3 parts needed in your hand
+    bput('put my powder in my small jar', 'You put')
+    EquipmentManager.new.empty_hands
+
+    walk_to(1977)
+    bput('look bough', 'The bough sways')
+    move('go path')
+    walk_to(12133)
+    case bput('get petal', 'You gather', 'You decide to leave')
+    when 'You decide to leave'
+      echo('You have gotten petals too recently, retry in 2 hours.')
+      exit
+    end
+    bput('put my petal in my mortar', 'You put')
+    bput('get my mortar', 'You get')
+    bput('get my pestle', 'You get')
+    while bput('crush petal in my mortar with my pestle', 'into some rose pulp', 'with your pestle') == 'with your pestle'
+      pause 1
+    end
+    EquipmentManager.new.empty_hands
+    bput('get pulp from my mortar', 'You get')
+    bput('put my pulp in my small jar', 'You put')
+    bput('get my small jar', 'You get')
+    bput('get my mixing stick', 'You get')
+    while bput('mix my small jar with my mix stick', 'combining them to create some rose attar', 'but nothing happens') == 'but nothing happens'
+      pause 1
+    end
+    EquipmentManager.new.empty_hands
+    case bput('get rose', 'You carefully', 'The rose thorns jab your hands and prevent you from getting another')
+    when 'The rose thorns jab your hands and prevent you from getting another'
+      echo('There is a 24 hour cooldown on getting a new rose, try again later.')
+      exit
+    end
+    bput('get attar from small jar', 'You get')
+    bput('sprinkle attar on my rose', 'You sprinkle')
+    bput('commune glythtide', 'You whisper a prayer over the rose')
+    walk_to(11691)
+    case bput('put rose on rock', 'You reverently place', 'You stop as you realize there')
+    when 'You stop as you realize there'
+      echo('Remove whatever is on the rock then put your rose on it, and offer rose on rock to continue.')
```

Comment:
What do you think about putting some quotes around `offer rose on rock` - something to indicate that's the _game command_? On the other hand, maybe it's already clear enough as is...